### ISSUE TITLE
Spinner.qml does not allow going through the list backwards

### ISSUE THREAD

**Issue by ghost** [iBelieve/ubuntu-ui-extras#3]
I am not sure if this is a limitation imposed by ListView, but I have hacked around in the file to try to get it not to crash.
 onValueChanged: {
        if (value >0){listView.currentIndex = value - minValue}
        else if(value<0){listView.currentIndex = listView.count - value}
        else if(value===0){listView.currentIndex=0}
    }
and I think this should (in theory) work... but it does not acquire the correct information
(the Bible verses are not displayed, though the correct books/chapter amounts are)
Basically if (value <=0) {CRASH} in the current state.
Am I missing something?

**Comment by iBelieve** [comment]
I think the reason is that my Spinner implementation is quite buggy :(

I was using it for a DatePicker and I was having some issues, so I ended up switching to the built in SDK DatePicker and created a custom style for it to make it look nicer. (See [here](https://plus.google.com/117574309170420884411/posts/VCRa33ZTipV)).

I wonder if there's a component in the SDK that would work for selecting Bible verses, not just dates? I'll investigate.

**Comment by ghost** [comment]
Picker is the correct thing to use.  I already pushed the change in uBible.
You should check it out, and clean it up if there is a better way of doing it.  It doesn't fit the style yet... but the implementation functions quite well for me.
